README.md

```diff
@@ -1,32 +1,3 @@
-<a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
-<!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
```

Comment:
- [ ] Kindly remove the following migration files, as they are empty and the last one is repetition that blocks the setting up of your project.

  ![ireow](https://user-images.githubusercontent.com/81468675/216367695-ab047a41-513b-40f7-a1e9-808be85a3de5.jpg)